service/src/main/java/bio/terra/landingzone/stairway/flight/CromwellStepsDefinitionProvider.java

```diff
@@ -22,14 +22,16 @@
 import bio.terra.landingzone.stairway.flight.create.resource.step.GetManagedResourceGroupInfo;
 import bio.terra.stairway.RetryRule;
 import bio.terra.stairway.Step;
+import com.azure.resourcemanager.AzureResourceManager;
 import java.util.List;
 import org.apache.commons.lang3.tuple.Pair;
 
 public class CromwellStepsDefinitionProvider implements StepsDefinitionProvider {
   // TODO: this doesn't take into account versioning
   @Override
   public List<Pair<Step, RetryRule>> get(
-      ArmManagers armManagers,
+      ArmManagers lzArmManagers,
+      AzureResourceManager adminSubResourceManager,
```

Comment:
Is this AzureResourceManager different from what we already have in ArmManagers?

Reply:
After further thought, I've scrapped this approach and shifted to config'ing out the storage account IDs rather than the dynamic queries against Azure to find them. I think this is simpler (if a bit more config to maintain).